### ISSUE TITLE
"No Known Servers" error when joining call

### ISSUE THREAD

**Issue by robertlong** [vector-im/matrix-video-chat#101]
When trying to join an existing call an error message "No Known Servers" can sometimes appear. I think this is a federation bug when joining a room on another homeserver.

It can be reproduced in this room:
https://matrixvoip.dev/#/room/!sxNfvhRkuQVHZJdgcK:matrix.org

**Comment by robertlong** [comment]
From @ara4n:
>Need to support ?via params on room IDs so you can find them over federation

**Comment by robertlong** [comment]
This is now implemented and that room may be accessed via this link:

https://matrixvoip.dev/#/room/!sxNfvhRkuQVHZJdgcK:matrix.org?via=matrix.org

I'm not sure how we're going to add those `via` parameters though. @ara4n sorry I'm still confused on what you had in mind here. matrix.to could definitely generate that link. I also added support for room aliases now which don't need the via parameter. But if we have a room list on the home page, how would I grab the `via` servers? Or am I still thinking about this wrong?
